code/src/index.css

```diff
@@ -1,13 +1,88 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Mono&display=swap');
+
 body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
+  margin: 20px;
+  font-family: 'Roboto Mono', monospace;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  font-size: 16px;
+  background: #fff;
+  display: flex;
+  justify-content: center;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.date {
+  font-style: italic;
+  font-size: 12px;
+  color: #4d4d4d;
+  float: right;
+}
+
+form {
+  height: 200px;
+  width: 500px;
+  padding: 15px;
+  border: 2px solid #4d4d4d;
+  background: #f2f0f0;
+  box-shadow: 6px 7px;
+  box-sizing: border-box;
+  
+  
+}
+form textarea {
+  margin-top: 15px;
+  padding: 5px
+
+}
+
+.form-text {
+  box-sizing: border-box;
+  width: 100%;
+  margin: 10px 0;
+  padding: 15px;
+  resize: none;
+}
+
```

Comment:
.form-text doesn't seem to be applies in your html. The styles should be applied to form textarea, I guess :)